Lib/importlib/_bootstrap_external.py

```diff
@@ -270,6 +270,7 @@ def _write_atomic(path, data, mode=0o666):
 #                         comprehensions #35224)
 #     Python 3.8b2  3412 (Swap the position of positional args and positional
 #                         only args in ast.arguments #37593)
+#     Python 3.9a0  3413 (add ASSERT_RAISE #34880)
```

Comment:
I think it is worth to increase the magic number more with a new version. To 3420.

Reply:
Okay. Done.